### PR TITLE
Limit permission to get files only from the UploadsS3Bucket

### PR DESCRIPTION
**Pull request by brunomoutinho**

### CODE REVIEW

File: cloudformation-template.json [hunk 1]
```diff
@@ -130,7 +130,7 @@
 									"s3:GetObject"
 								],
 								"Resource": [
-									"arn:aws:s3:::*"
+									{"Fn::Sub" : "arn:aws:s3:::${UploadsS3Bucket}"}
```
Comment (brunomoutinho) [inline cloudformation-template.json]: Haven't actually tested this yet... Will update with a comment as soon as I do this.

Comment (brunomoutinho) [reply cloudformation-template.json]: Ok, it does get the correct value for the bucket :)

Comment (joehoyle) [reply cloudformation-template.json]: mmm does this need a `*` at the end? I had assumed that a `GetObject` call for `s3://test-bucket/path/to/file.jpg` will need have to match a ARN like `arn:aws:s3:::test-bucket/*`

Comment (brunomoutinho) [reply cloudformation-template.json]: Sure! Totally missed this 😅

**Comment by brunomoutinho** [comment]
I also created the parameter for PrivateS3Bucket, so that the environment variable will be automatically created.

**Comment by brunomoutinho** [comment]
@joehoyle would you take a look at this after the update I did?
Also, do you know when a new version will be issued after these updates?

**Comment by joehoyle** [comment]
@brunomoutinho thanks! Periodically we make betas and stable releases as needed. I think with these changes we are ready to release a new minor version. Will follow up with a beta release shortly.

**Pull request merged**